frontend/src/containers/ViewDashboard.tsx

```diff
@@ -47,9 +45,7 @@ function ViewDashboard() {
       })}
     </>
   ) : (
-    <FooterLayout>
-      <FourZeroFour />
-    </FooterLayout>
+    <Redirect to="/404/page-not-found" />
```

Comment:
Now, when we redirect in the public website, the FourZeroFour component does not have footer:

https://fdingler.badger.wwps.aws.dev/404/page-not-found

This can be fixed changing the component to withPublicLayout(FourZeroFour) in the 404/page-not-found route

And at the end of the switch add:

<Redirect from='*' to='404/page-not-found' />

Reply:
The Redirect part in App.tsx is for completeness but only changing this: withPublicLayout(FourZeroFour) will fix the issue.

Reply:
Does the footer apply only to Admin screens? Or also to Public screens?